src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Extensions/IServiceCollectionExtensions.cs

```diff
@@ -142,6 +143,7 @@ public static void AddTextDocumentServices(this IServiceCollection services)
         services.AddRegisteringHandler<RazorDidChangeTextDocumentEndpoint>();
         services.AddHandler<RazorDidCloseTextDocumentEndpoint>();
         services.AddHandler<RazorDidOpenTextDocumentEndpoint>();
+        services.AddHandler<RazorDidSaveTextDocumentEndpoint>();
```

Comment:
It kinda doesn't surprise me this wasn't here.. since we dont advertise that we support it, and the endpoint does nothing. Do you have any context on this?

Doesn't need to be fixed in this PR of course.

Reply:
I have no context on this. I probably would have left it as is if the endpoint DID do anything 😝.